### PR TITLE
Fix Xcode's interactive error highlighting

### PR DESCRIPTION
**Pull request by wilhuff**

### CODE REVIEW

File: Firestore/Example/Firestore.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2411,7 +2411,7 @@
 				"${PODS_ROOT}/Target Support Files/Pods-Firestore_Example_tvOS/Pods-Firestore_Example_tvOS-frameworks.sh",
 				"${BUILT_PRODUCTS_DIR}/BoringSSL-GRPC-tvOS/openssl_grpc.framework",
 				"${BUILT_PRODUCTS_DIR}/GTMSessionFetcher-tvOS/GTMSessionFetcher.framework",
-				"${BUILT_PRODUCTS_DIR}/GoogleUtilities-Environment-Logger-tvOS/GoogleUtilities.framework",
+				"${BUILT_PRODUCTS_DIR}/GoogleUtilities-85d94889/GoogleUtilities.framework",
```
Comment (mikelehen) [inline Firestore/Example/Firestore.xcodeproj/project.pbxproj]: I don't know what caused this diff, but presumably it's intentional / correct...?

**Comment by paulb777** [comment]
Is there something about this issue that makes it specific to Firestore - or should CocoaPods automatically set the HEADER_SEARCH_PATHS like this?

File: Firestore/Example/Firestore.xcodeproj/project.pbxproj (hunk 1 above)
Comment (wilhuff) [reply Firestore/Example/Firestore.xcodeproj/project.pbxproj]: The latest changes to Firebase 5.20/5.20.1 cause this when you `pod install`. I'm just committing the results of that.

**Comment by wilhuff** [comment]
@paulb777 I'm not sure if this is generally applicable. This comes from the fact that there's no one right way to structure your `#import`/`#include`s. 

We settled on using unqualified quotes imports of our public headers from other public headers because that's what Cocoapods generates in the umbrella header. If we use anything else that defeats the way `#import` detects duplicates (which is apparently based on the string representing the include path rather than the actual file). I'm not sure this is universally the way it's meant to work.

**Pull request merged**